### ISSUE TITLE
Connecting to SqlServer errors

### ISSUE THREAD

**Issue by mamqua** [dotnet/maui#10111]
### Description

I tried with a lot of different solutions but nothing seems working: the android emulator can't connect sqlserver.
Is there some bug that should be fixed? i'm loosing time?

### Steps to Reproduce

```c#
string str = StringaConnessione(DB, Server);
                string qs = "SELECT * FROM dbo.Agenti;";
                using (SqlConnection connection = new SqlConnection(
                           str))
                {
                try
                {
                    connection.Open();

                }
                catch (Exception rt)
                {
                    string ff;
                    ff = rt.Message;
                }


                    SqlCommand command = new SqlCommand(qs, connection);
                
                    
                    command.ExecuteNonQuery();
                }
```

### Link to public reproduction project repository

no public repository

### Version with bug

6.0.486 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android from 29

### Did you find any workaround?

no

**Comment by drasticactions** [comment]
1. I doubt this is a MAUI (The UI Framework) issue, but another issue with your code. If you try to do the same thing in a net6 android app (`dotnet new android`) does it connect?
2. You _probably_ don't want to connect directly to a database through your app.

**Comment by janseris** [comment]
It's the uswdMAUI Android SSL implementation issue on some Androids.

**Comment by mamqua** [comment]
> 1. I doubt this is a MAUI (The UI Framework) issue, but another issue with your code. If you try to do the same thing in a net6 android app (`dotnet new android`) does it connect?
> 2. You _probably_ don't want to connect directly to a database through your app.

1. The code is really simple , it works if i use windowsmachine emulator but not with all android's emulators or android device conneccted
2.  i want connect directly to the database

**Comment by janseris** [comment]
Could you post what error do you receive and on which Android version?

**Comment by mamqua** [comment]
A connection was successfully established with the server, but then an error occurred during the pre-login handshake. (provider: SSL Provider, error: 31 - Encryption(ssl/tls) handshake failed)

since android 10 version in MAUI

**Comment by janseris** [comment]
@mamqua  are you using SQL Server Express 2019?

**Comment by mamqua** [comment]
> @mamqua are you using SQL Server Express 2019?

no but i tryed with all other version. sqlserver 2016 in the our network, sqlserver 2019 developr locally in my machine

**Comment by janseris** [comment]
https://github.com/dotnet/SqlClient/issues/1656
https://github.com/dotnet/SqlClient/issues/1662

**Comment by mamqua** [comment]
> Hi @mamqua. We have added the "s/needs-info" label to this issue, which indicates that we have an open question for you before we can take further action. This issue will be closed automatically in 7 days if we do not hear back from you by then - please feel free to re-open it if you come back to this issue after that time.

https://github.com/mamqua/TestMauiSqlServerConnection

**Comment by mamqua** [comment]
> [dotnet/SqlClient#1656](https://github.com/dotnet/SqlClient/issues/1656) [dotnet/SqlClient#1662](https://github.com/dotnet/SqlClient/issues/1662)

still tryed all but not working. atm i found a video with new tips i'll try follow that way

**Comment by mamqua** [comment]
> @mamqua could you share the video? I am interested as well to see any help with this issue

Now we are working at sqlserver configuration and updates but when we will finish that job i'll make a video. thx

**Comment by jfversluis** [comment]
As all linked issues show, I don't believe this is something that originates in .NET MAUI unfortunately. Keep tracking the linked issues to get this resolved. Another workaround, as already mentioned, is to not connect to a DB directly and place an API in-between.

**Comment by janseris** [comment]
@jfversluis It's an issue with Android TLS/SSL provider which MAUI uses. The errors don't happen on other platforms than MAUI. This means that it is a MAUI problem. Or a leftover Xamarin problem. 
How do you suggest that these issues should be solved when it doesn't happen for other platforms and it is evident that it's .NET Android problem if you check the exception trace that the error is from Android TLS/SSL provider?

**Comment by drasticactions** [comment]
"It's an issue with Android TLS/SSL provider which MAUI uses"

MAUI (This Repo) is a UI Framework for cross platform applications. MAUI (The UI Framework, this Repo) is built on top of MAUI (Or more specifically, .NET For Android), the .NET binding for Android.

Your issue is not something that can be fixed in this repo.

This issue should probably go to https://github.com/xamarin/xamarin-android or maybe https://github.com/dotnet/runtime, although it could be a root Android issue (Hence why it only happens on Android, and no other platform that MAUI (The UI Framework) supports, since those run on... other platforms... 

@jonathanpeppers Would you have any ideas of where this should go? Should it stay here?

**Comment by janseris** [comment]
Thank you.

>although it could be a root Android issue (Hence why it only happens on Android, and no other platform that MAUI (The UI Framework) supports, since those run on... other platforms...

Just to add a bit of additional info here, we have old Android applications (running on Android 8.1 and lower) communicating with the same SQL Server with TLS (TLS is always used for login even when Encrypt=false). These apps connect fine.
